datamol/convert.py

```diff
@@ -180,7 +180,44 @@ def to_inchi(mol: Union[str, Chem.rdchem.Mol]) -> Optional[str]:
     return Chem.MolToInchi(mol)
 
 
-def to_smarts(mol: Chem.rdchem.Mol) -> Optional[str]:
+def to_inchi_non_standard(
+    mol: Union[str, dm.Mol],
+    fixed_hydrogen_layer: bool = True,
+    undefined_stereocenter: bool = True,
+) -> Optional[str]:
+    """Convert a mol to a non-standard Inchi.
+
+    **Warning**: this function will return a **non-standard** Inchi. See
+    https://www.inchi-trust.org/technical-faq-2 for details.
+
+    It's important to not mix standard and non-standard InChi. If you don't know
+    much about non-standard InChi, we highly recommend you to use the
+    standard InChi with `dm.to_inchi()`.
+
+    Args:
+        mol: a molecule.
+        fixed_hydrogen_layer: whether to include a fixed hydrogen layer (`/FixedH`).
+        undefined_stereocenter: whether to include an undefined stereocenter layer (`/SUU`).
+    """
+
+    if mol is None:
+        return None
+
+    if isinstance(mol, str):
+        mol = dm.to_mol(mol)
+
+    options = ""
+
+    if fixed_hydrogen_layer:
+        options += "/FixedH"
+
+    if undefined_stereocenter:
+        options += " /SUU"
```

Comment:
looking a bit more closely at the inchi api docs, some options are "standard" like `AuxNone` and some are not, but all those mentionned above are non-standard

Reply:
You're right, I only used the options that was useful to us. It would be straightforward to add more as needed without breaking backward compat of the API.

So, I'll let it for later.

Reply:
There are a couple of tautomerism related options that could be included, actually.

Reply:
I have added 3 more options turned True by default. We can add more later if needed. Keep in mind that non-standard inchi should only be used for very specific needs.